aip/general/0162.md

```diff
@@ -66,9 +66,9 @@ message BookRevision {
 - The `message` name **must** be named `{ResourceType}Revision`.
 - The resource revision **must** contain a field with a message type of the
   parent resource, with a field name of `snapshot`.
-    - The value of `snapshot` **must** be the configuration of the parent
```

Comment:
I think the indentation was intentional here.

Reply:
Yes! I took away one space to keep it indented - if you look at the screenshot, it wasn't rendering as a sub-bullet - it was just showing inline as a hyphen - my hope is that it renders properly on the site because this seems to match what other sub bullets on this page had

Reply:
Oh I see, thanks for clarifying.